share/models/meta.py

```diff
@@ -26,6 +26,7 @@ class Award(ShareObject):
     award = ShareURLField(blank=True)
     description = models.TextField(blank=True)
     url = ShareURLField(blank=True)
+    funder = ShareManyToManyField('Funder', through='ThroughAwardFunders')
```

Comment:
Thinking out loud here, should this just be a generic sponsor that could be any entity?